code/game/objects/items/RPD.dm

```diff
@@ -29,6 +29,7 @@ GLOBAL_LIST_INIT(atmos_pipe_recipes, list(
 		new /datum/pipe_info/pipe("Injector", /obj/machinery/atmospherics/components/unary/outlet_injector, TRUE),
 		new /datum/pipe_info/pipe("Scrubber", /obj/machinery/atmospherics/components/unary/vent_scrubber, TRUE),
 		new /datum/pipe_info/pipe("Unary Vent", /obj/machinery/atmospherics/components/unary/vent_pump, TRUE),
+		new /datum/pipe_info/pipe("Dual-Port Vent", /obj/machinery/atmospherics/components/binary/dp_vent_pump, TRUE),
```

Comment:
DP-Pumps are missing an item form, which makes them non-deconstructable and results in no directionality selector in the UI.